src/index.mjs

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-process-exit */
+/* eslint no-console: 'off' */
```

Comment:
off topic

maybe we could add `"assert", "debug", "info", "exception"` to https://github.com/stylelint/eslint-config-stylelint/blob/4475504c22be754d30e529edfb7ad5db5765adf9/.eslintrc.js#LL37C19-L37C19

Reply:
Because we should detect debug messages or assertions with `console`, I think keeping it as-is is reasonable.